app/src/docs/v1.api-spec.yaml

```diff
@@ -15,19 +16,27 @@ servers:
 security:
   - OpenID: []
 paths:
-  /checks/status:
+  /health:
     get:
       summary: Returns status of correspondent APIs
       operationId: getChecksStatus
       tags:
-        - Checks
+        - Health
       responses:
         '200':
-          description: Responds if the correspondent API endpoint(s) are healthy
+          description: Returns the healthiness of the correspondent API endpoint(s)
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/EndpointList'
+                type: object
+                required:
+                  - endpoints
+                properties:
+                  endpoints:
+                    type: array
+                    description: A list of Endpoint Statuses
```

Comment:
Let's do a cleanup of the nomenclature after the spec itself is approved...

Reply:
updated the names to reference Health object instead of EndpointStatus.